packages/server/src/mail/mail-services.controller.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Controller,
+  Get,
+  Param,
+  Patch,
+  UseGuards,
+  HttpException,
+  HttpStatus,
+  Body,
+} from '@nestjs/common';
+import { MailService } from './mail.service';
+import { JwtAuthGuard } from 'guards/jwt-auth.guard';
+import { MailServiceModel } from './mail-services.entity';
+import { MailServiceWithSubscription } from '@koh/common';
+import { User } from 'decorators/user.decorator';
+import { UserModel } from 'profile/user.entity';
+import { UserSubscriptionModel } from './user-subscriptions.entity';
+import { EmailVerifiedGuard } from 'guards/email-verified.guard';
+@UseGuards(JwtAuthGuard, EmailVerifiedGuard)
+@Controller('mail-services')
+//handles notfication settings for emails
+export class MailServicesController {
```

Comment:
No protection from unauthorized users

Reply:
UseGuards(JwtAuthGuard, EmailVerifiedGuard) gives protection for unauthorized users? Also these endpoints use `@User` so they only grab the currently logged in user's data.